awx_collection/tests/integration/targets/workflow_job_template/tasks/main.yml

```diff
@@ -524,7 +518,6 @@
       ansible.builtin.assert:
         that:
           - no_unified_results is defined
-          - no_unified_results.failed
```

Comment:
Removing this in other cases seems fine, because the other assertions are more strict. But this case seems more problematic, and may give false-positives.